openapi/nexus.json

```diff
@@ -9130,6 +9185,94 @@
           "switch_histories"
         ]
       },
+      "AllowedSourceIps": {
+        "description": "Allowlist of IPs or subnets that can make requests to user-facing services.",
+        "type": "object",
+        "properties": {
+          "allowed_ips": {
+            "description": "The allowlist of IPs or subnets.",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/AllowedSourceIps2"
+              }
+            ]
+          },
+          "time_created": {
+            "description": "Time the list was created.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "time_modified": {
+            "description": "Time the list was last modified.",
+            "type": "string",
+            "format": "date-time"
+          }
+        },
+        "required": [
+          "allowed_ips",
+          "time_created",
+          "time_modified"
+        ]
+      },
+      "AllowedSourceIps2": {
```

Comment:
We get "AllowedSourceIps2" because we're using the same name for a type twice. I'd suggest changing one or both to append "Update" or "View" or "Body" or whatever.